packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -652,6 +653,7 @@ export const PersonalDetailsForm = ({
                                                         disabled={!isChangeableField('first_name')}
                                                         error={errors.first_name}
                                                         id={'first_name'}
+                                                        data-testid='first_name'
```

Comment:
```suggestion
                                                        data-testid='dt_first_name'
```
Use `dt` in front of every data-testid declaration